assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/components/NoPricesBanner.tsx

```diff
@@ -4,11 +4,19 @@ import { __ } from '@wordpress/i18n';
 import { Banner } from '@application/ui/display';
 
 import AddDefaultPricesButton from '../buttons/AddDefaultPricesButton';
+import { TicketPriceCalculatorProps } from './TicketPriceCalculator';
 
-const NoPricesBanner = () => {
-	const description = __(
-		'Click the button below to load your default prices into the calculator. Additional ticket price modifiers can be added or removed. Click the save button below to assign which dates this ticket will be available for purchase on.'
+interface Props extends Pick<TicketPriceCalculatorProps, 'context'> {}
+
+const NoPricesBanner: React.FC<Props> = ({ context }) => {
+	let description = __(
+		'Click the button below to load your default prices into the calculator. Additional ticket price modifiers can be added or removed.'
 	);
+
+	if (context === 'futureUseCase') {
```

Comment:
### 😆  LOLZ 😆

when I wrote `futureUseCase`  in my suggestion, I meant that to represent some other use case that may arise in the future !
that was not intended to be something added to the code now.

So for this particular use case, you might call it `editTicketForm` since the context is the TPC inside of the edit ticket form stepper

Reply:
LOL, was distracted by that VS Code feature and made this silly mistake;
updated

Reply:
ya that's kinda what I figured 😄